provisioner/terraform/install.go

```diff
@@ -64,13 +64,24 @@ func Install(ctx context.Context, log slog.Logger, dir string, wantVersion *vers
 		Version:    TerraformVersion,
 	}
 	installer.SetLogger(slog.Stdlib(ctx, log, slog.LevelDebug))
-	log.Debug(
-		ctx,
-		"installing terraform",
-		slog.F("prev_version", hasVersionStr),
-		slog.F("dir", dir),
-		slog.F("version", TerraformVersion),
-	)
+
+	logInstallCtx, logInstallCancel := context.WithCancel(ctx)
+	go func() {
+		// If the installation is taking too long, log a notice.
+		select {
+		case <-time.After(10 * time.Second):
+			log.Info(
+				logInstallCtx,
+				"installing terraform",
```

Comment:
At least for external provisioners, I'd like this to always be logged. For internal, perhaps we can have a different log, like "installing terraform is taking a while.. still installing" (not too bothered about the exact copy, just trying to get the idea across).